development-guidelines/non-standard-tokens.md

```diff
@@ -0,0 +1,56 @@
+# Known non-standard ERC20 tokens
+
+The following tokens are known to be non-standard ERC20 tokens. They may have additional risks that must be covered.
+
+## Missing Revert
+
+These tokens do not revert when a transfer fails due to missing funds. Protocols that integrate these tokens must include a check for the transfer success status in the function's return value and handle the failure case appropriately.
```

Comment:
May be worth highlighting specifically that these tokens return booleans that represent success/failure, which is why the return value has to be checked